thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -29,6 +29,16 @@ struct TDataNodeRegisterReq {
   2: optional map<string, TStorageGroupSchema> statusMap
 }
 
+struct TDataNodeRemoveReq {
```

Comment:
`TRemoveDataNodeReq`

Reply:
I keep it format with TDataNodeRegisterReq